app/statistics/server/functions/get.js

```diff
@@ -71,10 +77,16 @@ statistics.get = function _getStatistics() {
 	statistics.totalUsers = Meteor.users.find().count();
 	statistics.activeUsers = Meteor.users.find({ active: true }).count();
 	statistics.nonActiveUsers = statistics.totalUsers - statistics.activeUsers;
-	statistics.onlineUsers = Meteor.users.find({ statusConnection: 'online' }).count();
-	statistics.awayUsers = Meteor.users.find({ statusConnection: 'away' }).count();
-	statistics.totalConnectedUsers = statistics.onlineUsers + statistics.awayUsers;
-	statistics.offlineUsers = statistics.totalUsers - statistics.onlineUsers - statistics.awayUsers;
+	statistics.onlineUsers = Meteor.users
+		.find({ statusConnection: 'online' })
+		.count();
+	statistics.awayUsers = Meteor.users
+		.find({ statusConnection: 'away' })
+		.count();
+	statistics.totalConnectedUsers =
+		statistics.onlineUsers + statistics.awayUsers;
+	statistics.offlineUsers =
+		statistics.totalUsers - statistics.onlineUsers - statistics.awayUsers;
```

Comment:
are all of the second line things something your IDE is doing?

Reply:
Yeah, I had an issue with my VSCode, I'm pushing a new version